Add missing parts of documentation (#172)
kyma-incubator/hack-showcase: docs/github-connector/installation.md

### Prerequisites

- Personal Access Token with desired privileges. To generate a new token, go [here](https://github.com/settings/tokens) or access **Personal access tokens** in the account through **Developer settings** in **Settings**. 

> > **NOTE**: In case of generating token for an organization the [OAuth or Github App](https://developer.github.com/apps/) is needed.

Reviewer: ```suggestion
> **NOTE**:To generate a token for an organization, you need the [OAuth or Github App](https://developer.github.com/apps/).
```
